crates/subspace-runtime/src/lib.rs

```diff
@@ -159,7 +160,24 @@ const SLOT_PROBABILITY: (u64, u64) = (1, 6);
 const GLOBAL_RANDOMNESS_UPDATE_INTERVAL: BlockNumber = 256;
 
 /// Number of slots between slot arrival and when corresponding block can be produced.
-const BLOCK_AUTHORING_DELAY: SlotNumber = 6;
+const BLOCK_AUTHORING_DELAY: SlotNumber = 4;
+
+/// Interval, in blocks, between blockchain entropy injection into proof of time chain.
+const POT_ENTROPY_INJECTION_INTERVAL: BlockNumber = 20;
+
+/// Interval, in entropy injection intervals, where to take entropy for injection from.
+const POT_ENTROPY_INJECTION_LOOKBACK_DEPTH: u8 = 20;
+
+/// Delay after block, in slots, when entropy injection takes effect.
+const POT_ENTROPY_INJECTION_DELAY: SlotNumber = 15;
+
+// Entropy injection interval must be bigger than injection delay or else we may end up in a
+// situation where we'll need to do more than one injection at the same slot
+const_assert!(POT_ENTROPY_INJECTION_INTERVAL as u64 > POT_ENTROPY_INJECTION_DELAY);
```

Comment:
This assertion ensures that parameters above make sense. Imagine we have interval 1 block and lookback 1 block, while delay is 5 slots.

In this case it is possible that we have more than one entropy injection known and scheduled at a time, which requires us to expose it in all APIs.

While we can do that if required, it is not required with current parameters and this assertion ensures it is not possible to configure the chain such that we end up in that situation, it will fail to compile, simplifying implementation.